site/public/js/solution-ta-notes.js

```diff
@@ -1,5 +1,5 @@
-/* global buildCourseUrl, displaySuccessMessage, displayErrorMessage, csrfToken */
-/* exported updateSolutionTaNotes, showSolutionTextboxCont, cancelEditingSolution */
+/* global buildCourseUrl, displaySuccessMessage, displayErrorMessage, csrfToken, previewMarkdown */
+/* exported updateSolutionTaNotes, showSolutionTextboxCont, cancelEditingSolution, previewSolutionNotesMarkdown */
```

Comment:
You deleted some functions so you should probably delete them from the comment too. I'm curious for my own knowledge -- do these comments do anything?